### ISSUE TITLE
Adminarea/forums forum position by typing in the number (now only by click on arrow)

### ISSUE THREAD

**Issue by Noir1992** [script-solution/Boardsolution#80]
If you have many forums (eg. 40) and you create a new one, it will be at the last position. If you want to have it at positon 3 you need to click many many times on the arrow to get it on that position. It would be nice to have the option to type the number directly in, so the new forum gets the position 3, the old number 3 become number 4 ...

**Comment by Rafioso** [comment]
I agree with that, but really, how often do you insert new forums? Is this really necessary.
